keps/sig-ui/5008-kep-template/README.md

```diff
@@ -0,0 +1,948 @@
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+The proposal is to ship the Headlamp desktop experience as a new Kubernetes
+Desktop and the Headlamp in-cluster UI to eventually become the next generation
+of the Kubernetes Dashboard.
+
+Effectively this entails moving the Headlamp project under the Kubernetes
+GitHub organization and under the auspices of Kubernetes SIG UI. To be
+specific, we would propose that the Headlamp project be moved to a repo under
+the Kubernetes project; something to the effect of kubernetes/ui.
+The current repo kubernetes/dashboard would remain but include a note about the
+existence of the other repo. This would allow for users to compare the projects
+and make an informed decision about which works best for their use case. Over
+time, we can measure adoption trends and make decisions based on that data.
+
+The Headlamp branding would be phased out.
+
+### User Stories
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+One of Headlamp's strengths is that it was designed to both deliver a great
+end-user experience and be a platform for others to build upon and integrate
+with. This is demonstrated by the wide range of user stories it enables and one
+of the reasons we believe it can become a hub for the community.
+
+_**Note:** Using “Headlamp” in the following examples to avoid confusion._
+
+#### Story 1
+
+As an end user I want to get an overview of my cluster. I start the Headlamp
+Desktop app which picks up my clusters from my kubeconfig and I’m now able to
+explore my clusters.
+
+#### Story 2
+
+As a new Kubernetes user I want to learn more about Kubernetes. I use the
+Headlamp Desktop app to guide me in installing a local kubernetes cluster using
+minikube. Once my local cluster is installed I’m presented with a tutorial
+plugin which guides me through the various aspects of the UI and Kubernetes
+concepts.
+
+_**Note:** We have the minikube installer plugin in POC but the tutorial plugin
+is not something we currently have._
+
+#### Story 3
+
+As a cluster admin I was to provide a user interface for my users. I choose
+which plugins I want to include for my users and deploy Headlamp to my cluster.
+I have peace of mind knowing that Headlamp respects the RBAC rules that each
+user has.
+
+#### Story 4
+
+As a vendor I want to create a new Kubernetes user interface for my system. I
+take Headlamp Base and build a set of plugins and take some of the existing
+plugins to create my custom UI. I customize the UI with my company/product
+branding and provide that to my users with or without them knowing it’s
+Headlamp. If I desire, I can even allow my users to extend my customer UI using
+the same way I did.
```

Comment:
Can this be done right now with the existing Headlamp? If so, maybe we should mention that.

Reply:
Yes, this is possible right now (all user stories are possible ATM, except when indicated by notes, like the minikube plugin being a POC still). How can we indicate that we do support these more explicitly?

Reply:
@sftim any suggestions or should it be kept as is?  The description and branding possibilities are clear to me.